docs_src/hardwaresetup.md

```diff
@@ -1,52 +1,41 @@
-# Setup for Hardware
+# Set up Hardware
 
-## 11th & 12th Gen Intel® Core™ Setup
+## Set up 11th Gen Intel® Core™ Processor and 12th Gen Intel® Core™ Processor
 
-### Step 1: Install Ubuntu 20.04
```

Comment:
Please do check the version of Ubuntu. Is it different for different processors? If the version is the same, I suggest combining the steps for 11th and 12th Gen and Xeon as it is a repetition. You can add Xeon to the heading.